src/anaconda/install.sh

```diff
@@ -47,7 +47,15 @@ elif [ "${USERNAME}" = "none" ] || ! id -u ${USERNAME} > /dev/null 2>&1; then
 fi
 
 architecture="$(uname -m)"
-if [ "${architecture}" != "x86_64" ]; then
+if [ "${architecture}" == "amd64" ]; then
+    CONDA_ARCH="x86_64"
+elif [ "${architecture}" == "x86_64" ]; then
+    CONDA_ARCH="x86_64"
+elif [ "${architecture}" == "arm64" ]; then
+    CONDA_ARCH="aarch64"
+elif [ "${architecture}" == "aarch64" ]; then
+    CONDA_ARCH="aarch64"
+else
     echo "(!) Architecture $architecture unsupported"
     exit 1
 fi
```

Comment:
```suggestion
case "${architecture}" in
        amd64) architecture=x86_64 ;;
        arm64) architecture=aarch64 ;;
        *)
            echo "Anaconda does not support machine architecture '$architecture'. Please use an x86-64 or ARM64 machine."
            exit 1
    esac
```

How about we simplify the code?

Reply:
happy to implement though as i read your suggestion it accomplishes a different goal. the goal is to support amd64, x86_64, arm64, and aarch64. and in the case of amd64 use the x86_64 URL and in the case of the arm64 use the aarch64 URL

Reply:
I was hoping to keep the goal in tact but didn't realize that we are using `architecture="$(uname -m)"` to compute `architecture`. Thanks for catching that! 

How about we use 👇 instead? That way I believe it only returns `amd64` of `arm64`. Hence, we'd skip another conditions. 

```
architecture=$(dpkg --print-architecture)
```

Reply:
nice that makes sense. i'll work up a commit.